changelog/4978.removal.rst

```diff
@@ -0,0 +1,9 @@
+Remove option ``use_cls_token`` from tokenizers and option ``return_sequence`` from featurizers.
+
+By default all tokenizer add a special token (``__CLS__``) to the end of the list of tokens.
+This token will be used to capture the features of the whole utterance.
+
+The featurizers will return a matrix of size (token-length x feature-dimension) by default.
```

Comment:
```suggestion
The featurizers will return a matrix of size (number-of-tokens x feature-dimension) by default.
```
I got confused initially if by `token-length` you meant length of an individual token. Suggested this for clarity.